base.go

```diff
@@ -221,10 +222,10 @@ func (a *Auth) request(serviceUrl ServiceUrl, action string, body interface{}, o
 	req.Header.Add("Content-Type", "text/xml;charset=UTF-8")
 	req.Header.Add("SOAPAction", action)
 	resp, err := a.Client.Do(req)
+	defer resp.Body.Close()
```

Comment:
I think this should stay in the original place, otherwise it's going to cause panic every time there's a request error: https://play.golang.org/p/kmFmviL43da

Reply:
Oh!! That's good to know. I've changed it back :)